### PR TITLE
OHRM5X-1340: Admin - Corporate Branding Screen

### PR DESCRIPTION
**Pull request by Super-Chama**

### CODE REVIEW

File: src/client/src/orangehrmAdminPlugin/index.ts [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+import EditModuleConfiguration from './pages/moduleConfiguration/EditModuleConfiguration.vue';
+import WorkShift from './pages/workShift/WorkShift.vue';
+import SaveWorkShift from './pages/workShift/SaveWorkShift.vue';
+import EditWorkShift from './pages/workShift/EditWorkShift.vue';
+import EmailSubscription from './pages/emailSubscription/EmailSubscription.vue';
+import EditEmailSubscription from './pages/emailSubscription/EditEmailSubscription.vue';
+import LocalizationConfiguration from './pages/localization/LocalizationConfiguration.vue';
+import CorporateBranding from './pages/corporateBranding/CorporateBranding.vue';
+
+export default {
```
Comment (RajithaKumara) [inline src/client/src/orangehrmAdminPlugin/index.ts:67]: :ok_hand:

File: src/client/src/orangehrmAdminPlugin/pages/corporateBranding/CorporateBranding.vue [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+            </oxd-grid-item>
+          </oxd-grid>
+        </oxd-form-row>
+
+        <oxd-form-row>
+          <oxd-grid :cols="3" class="orangehrm-full-width-grid">
+            <oxd-grid-item>
+              <div class="orangehrm-sm-field">
+                <oxd-text tag="p" class="orangehrm-sm-field-label">
+                  Social Media Images
```
Comment (RajithaKumara) [inline src/client/src/orangehrmAdminPlugin/pages/corporateBranding/CorporateBranding.vue:143]: :thinking:

**Review by RajithaKumara — APPROVED** [review]

File: src/client/src/orangehrmAdminPlugin/pages/corporateBranding/CorporateBranding.vue (hunk 2 above)
Comment (Super-Chama) [reply src/client/src/orangehrmAdminPlugin/pages/corporateBranding/CorporateBranding.vue]: updated

File: src/client/src/core/styles/overrides.scss [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+//
+// OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+// without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+// See the GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along with this program;
+// if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+// Boston, MA  02110-1301, USA
+
+@function darken($color, $amount) {
```
[the remark this answers is not in the document]
Comment (Super-Chama) [reply src/client/src/core/styles/overrides.scss:17]: oxd changes will added. this file will change

File: src/plugins/orangehrmCorePlugin/templates/vue.html.twig [hunk 4]
```diff
@@ -13,7 +13,19 @@
   <link href="{{ publicPath }}/dist/css/app.css?{{ assetsVersion }}" rel="stylesheet"/>
 {% endblock %}
 
+
 {% block body %}
+
+  <style>
+  :root {
```
[the remark this answers is not in the document]
Comment (RajithaKumara) [reply src/plugins/orangehrmCorePlugin/templates/vue.html.twig:20]: hope we are going to add these to `vue.html.twig` and other templates?

**Review by RajithaKumara — APPROVED** [review]

**Pull request merged**